### ISSUE TITLE
Improve Setup project documentation

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress-documentation#1555]
## I'm submitting a...

```
[ ] Bug report 
[x] Content update
[ ] Process update (build, deployment, ... )
```

## Type of bug / changes

On this section: https://docs.cypress.io/guides/core-concepts/dashboard-service.html#Setup

### Update this section to instead of just being the gif (which can be hard to follow), to walk through each step in photos. 

![](https://user-images.githubusercontent.com/1271364/54583909-15791880-4a44-11e9-9f4e-6487c71de113.png)
![](https://user-images.githubusercontent.com/1271364/54583901-1316be80-4a44-11e9-9ac4-95057e25b316.png)
![](https://user-images.githubusercontent.com/1271364/54583908-14e08200-4a44-11e9-9029-91762d2a690a.png)
![](https://user-images.githubusercontent.com/1271364/54583907-14e08200-4a44-11e9-8f5d-c42554703034.png)
![](https://user-images.githubusercontent.com/1271364/54583906-1447eb80-4a44-11e9-93e5-e096a1aadc30.png)
![](https://user-images.githubusercontent.com/1271364/54583905-1447eb80-4a44-11e9-84d3-c49fa6db2c5c.png)
![](https://user-images.githubusercontent.com/1271364/54583904-1447eb80-4a44-11e9-95bc-31f6643231c4.png)
![](https://user-images.githubusercontent.com/1271364/54583902-13af5500-4a44-11e9-83f0-cd327f4a9b49.png)

### Also add some possible error states. Maybe to error messages page???

![](https://user-images.githubusercontent.com/1271364/54583896-11e59180-4a44-11e9-802e-bab9f6e0760e.png)
![](https://user-images.githubusercontent.com/1271364/54583898-127e2800-4a44-11e9-9348-3a8565f1ae43.png)
![](https://user-images.githubusercontent.com/1271364/54583900-1316be80-4a44-11e9-8097-b3fb16b05962.png)

**Comment by flrnprz** [comment]
Hey there! I'd like to take this up.

**Comment by jennifer-shehane** [comment]
@flrnd Feel free please!

**Comment by jennifer-shehane** [comment]
Decided against this idea - it's a lot on the page. Should just rerecord a gif.
